client/my-sites/stats/stats-notices/all-notice-definitions.ts

```diff
@@ -130,4 +103,39 @@ const ALL_STATS_NOTICES: StatsNoticeType[] = [
 	},
 ];
 
+function shouldShowCommercialSiteUpgradeNotice( {
```

Comment:
Can we please not change the pattern here just to be consistent?

Not related to the PR: I'm thinking maybe we could put those notices back to their own files, where the notice component are defined. It'll better satisfy the open/close principal. Thought?